src/librustc_driver/lib.rs

```diff
@@ -1329,20 +1325,19 @@ pub fn diagnostics_registry() -> errors::registry::Registry {
     Registry::new(&all_errors)
 }
 
-pub fn get_args() -> Vec<String> {
```

Comment:
@nrc I assume you mean only this function? This seems very useful, I guess because of the use of `early_error` this can't be substituted with anything in `std` directly.